rdrf/rdrf/views/form_view.py

```diff
@@ -861,6 +861,14 @@ def _get_formlinks(self, user, context_model=None):
         else:
             return []
 
+    def _set_initial_randomized_data(self, form_class, initial_data):
+        if not initial_data:
+            initial_data = {}
+        for field_name, field in form_class.base_fields.items():
+            if hasattr(field, 'randomized_value'):
+                initial_data[field_name] = field.randomized_value
+        return initial_data
+
```

Comment:
`return {field_name: field.randomized_value for field_name, field in form_class.base_fields.items() if  hasattr(field, 'randomized_value'}`

reads better and simpler IMO.

Reply:
Updated.